src/Admin/Views/Organizations/_OrganizationForm.cshtml

```diff
@@ -0,0 +1,253 @@
+@using Bit.SharedWeb.Utilities
+@model OrganizationEditModel
+
+<form method="post" id="edit-form">
+    @if (Model.Provider != null)
+    {
+        @Html.HiddenFor(m => m.Provider.Id)
+        @Html.HiddenFor(m => m.Provider.Type)
+    }
+    
+    <input asp-for="SalesAssistedTrialStarted" type="hidden">
+    <h2>General</h2>
+    <div class="row">
+        <div class="col-sm">
+            <div class="form-group">
+                <label asp-for="Name"></label>
+                <input type="text" class="form-control" asp-for="Name" required>
+            </div>
+        </div>
+    </div>
+
+    @if (Model.Provider?.Type == ProviderType.Reseller)
+    {
+        <div class="row">
+            <div class="col-sm">
+                <div class="form-group">
+                    <label>Client Owner Email</label>
+                    @if (!string.IsNullOrWhiteSpace(Model.Owners))
+                    {
+                        <input type="text" class="form-control" asp-for="Owners" readonly="readonly">
+                    }
+                    else
+                    {
+                        <input type="text" class="form-control" asp-for="Owners" required>
+                    }
+                    <label class="form-check-label small text-muted align-top">This user should be independent of the Provider. If the Provider is disassociated with the organization, this user will maintain ownership of the organization.</label>
+                </div>
+            </div>
+        </div>
+    }
+    @if (Model.Organization != null)
+    {
+        <div class="form-check mb-3">
+            <input type="checkbox" class="form-check-input" asp-for="Enabled">
+            <label class="form-check-label" asp-for="Enabled"></label>
+        </div>
+    }
+    <h2>Business Information</h2>
+    <div class="row">
+        <div class="col-sm">
+            <div class="form-group">
+                <label asp-for="BusinessName"></label>
+                <input type="text" class="form-control" asp-for="BusinessName">
+            </div>
+        </div>
+    </div>
+    <h2>Plan</h2>
+    <div class="row">
+        <div class="col-sm">
+            <div class="form-group">
+                <label asp-for="PlanType"></label>
+                @{
+                    var planTypes = Enum.GetValues<PlanType>()
+                        .Where(p => Model.Provider == null || p is >= PlanType.TeamsMonthly and <= PlanType.EnterpriseAnnually)
+                        .Select(e => new SelectListItem
+                        { 
+                            Value = ((int)e).ToString(),
+                            Text = e.GetDisplayAttribute()?.GetName() ?? e.ToString()
+                        })
+                        .ToList();
+                }
```

Comment:
Is this change so that we can filter out old plan types? Is that a requirement? I can't see it in the Figma.

Note that the trial buttons at the bottom of the page don't work unless a Free plan is selected, which isn't possible at the moment.

This change also affects the Edit page, is that intentional?

Reply:
I decided to filter the Plan Types because only Teams and Enterprise are assignable to a Provider. I'll check with Product if this makes sense and also about removing the trial buttons.

Reply:
Priya left a comment in Jira saying we can just remove the trial buttons. I'll leave this thread unresolved so you can do that.

Reply:
Also, if a Reseller can only be associated with an Organization on a certain plan (Teams or Enterprise), then you should also enforce that somewhere in a service or command. Not just at the view level.

Reply:
I'll be sure to add that on EC-431